forward_engineering/helpers/typeHelper.js

```diff
@@ -134,7 +134,7 @@ const getStructuralTypeHandler = (type, isNeedToBeFrozen, udtTypeMap) => {
 			if (nestedType !== undefined) {
 				return nestedType;
 			} else {
-				('');
+				return '';
```

Comment:
nope, the `else` block needs to be omitted completely in this case, just leave the `if` condition

Reply:
done